components/tag-list/test/tag-list.test.js

```diff
@@ -73,10 +51,9 @@ describe('d2l-tag-list', () => {
 	describe('clearable', () => {
 		it('should dispatch expected events when Clear All clicked', async() => {
 			const elem = await fixture(clearableFixture);
-			await waitUntil(() => elem._items, 'List items did not become ready');
 			const button = elem.shadowRoot.querySelector('d2l-button-subtle.d2l-tag-list-clear-button');
 
-			setTimeout(() => button.click());
+			clickElem(button);
```

Comment:
I think we still want to do this kind of thing in a timeout in case the event gets fired synchronously from the click handler.

Reply:
I wasn't getting any flake locally without it, and I _think_ it might always be safe to do this without, but it'd be good to know definitively either way before we convert/write more of these. I'll look into it.

Reply:
Also, if it's not safe, maybe we wrap them so it is always safe?

Reply:
Yeah maybe it makes sense for us to put the `setTimeout` inside all of these utilities eh? The Promises they return can still resolve after the action has complete if that's what they're going for, but if you don't care you can just call it and then `await` an event on the next line.

Reply:
Looked into this and discussed offline. There are multiple steps in other processes that make these interactions inherently significantly delayed (in the event loop, not actually time) so for now we're thinking these should be safe to run in this order without intervention, but we can always build in a delay later if needed.